packages/react-components/MIGRATION-NOTES.md

```diff
@@ -0,0 +1,321 @@
+# Notes on migration to `@fluentui/react-components@9.0.0-rc.1`
```

Comment:
can we specify migration from what version are we talking? ( based on previous convo it should be from Beta to RC)

Reply:
```suggestion
# Migration notes from `@fluentui/react-components@9.0.0-beta.5`to `@fluentui/react-components@9.0.0-rc.1`
```